cmake-proxies/portsmf/CMakeLists.txt

```diff
@@ -1,5 +1,5 @@
 
-add_library( ${TARGET} STATIC )
+add_library( ${TARGET} OBJECT )
```

Comment:
`STATIC` does not require `__declspec`, so I don't understand your motivation here. I don't think that the problem was identified correctly.

Reply:
I recall the Windows build failing when I undid that (already a few years ago).  I could repeat the experiment and see if the build still fails without it.